### PR TITLE
chore(php): add xsl extension

### PR DESCRIPTION
**Pull request by xXNoceboXx**

### CODE REVIEW

**Comment by mremi** [comment]
please update CHANGELOG.md

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Versions
 2022-02-28
 ----------
 
-* PHP: workaround for installing sockets and zip extensions
+* PHP: workaround for installing sockets, xsl and zip extensions
```
Comment (mremi) [inline CHANGELOG.md:12]: please add a new line, this is not the same issue

**Comment by mremi** [comment]
please rebase

**Comment by xXNoceboXx** [comment]
@mremi can you launch workflow, please ?

**Pull request merged**